### PR TITLE
TASK-53732: fixed consult winners list after click on show more button

### PR DESCRIPTION
**Pull request by GouadriaHanen**
while retrieving challenges after clicking on loadMore 
getAnnouncementsByChallenge takes the new offset and limit which may return a null list since the announcements number may not reach the offset

### CODE REVIEW

**Comment by ahamdi** [comment]
@GouadriaHanen What is the required number of announcements to load by default ? cc @mzorai

**Comment by mzorai** [comment]
@ahamdi already discussed with @GouadriaHanen and agreed to display 2 announcements by default

File: services/src/main/java/org/exoplatform/addons/gamification/rest/ChallengeRest.java [hunk 1]
```diff
@@ -180,8 +180,8 @@ public Response getAllChallengesByUser(@ApiParam(value = "Offset of result", req
       List<ChallengeRestEntity> challengeRestEntities = new ArrayList<>();
       for (Challenge challenge : challenges) {
         List<Announcement> challengeAnnouncements = announcementService.findAllAnnouncementByChallenge(challenge.getId(),
-                                                                                                       offset,
-                                                                                                       limit);
+                                                                                                       0,
+                                                                                                       10);
```
Comment (ahamdi) [inline services/src/main/java/org/exoplatform/addons/gamification/rest/ChallengeRest.java]: @GouadriaHanen If we need just 2 then no need to put 10 as limit c c @mzorai

Comment (GouadriaHanen) [reply services/src/main/java/org/exoplatform/addons/gamification/rest/ChallengeRest.java]: done!

**Review by ahamdi — APPROVED** [review]

**Pull request merged**